runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -58,6 +60,18 @@
 @Slf4j
 public class ItemManager
 {
+	private static final List<Pattern> normalizePatterns = new ArrayList<Pattern>()
+	{{
+		// cakes, pizzas, pies
+		add(Pattern.compile("^(?:[0-9]/[0-9]|half an?) ([^(0-9]+)|$"));
```

Comment:
[0-9]/[0-9] could probably be [1-9]/[1-9]? There's no items that are "0/3" for instance, and I doubt there will ever be 0 in the denominator either.